### ISSUE TITLE
ensure fresh install works even if ifadv missing

### ISSUE THREAD

**Issue by mdingemanse** [elpaco-escience/talkr#117]
Was just testing a new installation of `talkr` in an updated version of R (3.4.4). Building the package from github with devtools failed because it tries to build the vignette which assumes presence of `ifadv` which isn't there yet on the very first run.

@PabRod can you confirm/reproduce?
To reproduce:
1. fresh R version
2. fresh installation of Rstudio without any packages
3. run `install.packages("devtools")`
4. run `devtools::install_github("elpaco-escience/talkr", build_vignettes = TRUE)`

Produces this error:

```
── R CMD build ─────────────────────────────────────────────────────────────────────────────
✔  checking for file 'C:\Users\markd\AppData\Local\Temp\RtmpEl5gUg\remotes2ddc85a6197\elpaco-escience-talkr-73639d0/DESCRIPTION'
─  preparing 'talkr': (356ms)
✔  checking DESCRIPTION meta-information ...
─  installing the package to build vignettes
E  creating vignettes (5.7s)
   --- re-building 'workflows.Rmd' using rmarkdown
   
   Quitting from lines 35-38 [unnamed-chunk-2] (workflows.Rmd)
   Error: processing vignette 'workflows.Rmd' failed with diagnostics:
   there is no package called 'ifadv'
   --- failed re-building 'workflows.Rmd'
   
   SUMMARY: processing the following file failed:
     'workflows.Rmd'
   
   Error: Vignette re-building failed.
   Execution halted
Error: Failed to install 'talkr' from GitHub:
  ! System command 'Rcmd.exe' failed
```

**Comment by mdingemanse** [comment]
Can also confirm installation does work after first installing ifadv.

**Comment by PabRod** [comment]
TL;DR: `Remotes` are not allowed in CRAN.

In the CRAN guide, specifically in the [Devtools dependencies subsection](https://cran.r-project.org/web/packages/devtools/vignettes/dependencies.html), we can read the following:

> When you submit your package to CRAN, all of its dependencies must also be available on CRAN. For this reason, release() will warn you if you try to release a package with a Remotes field.

## Quick fix
Replace `ifadv::ifadv` calls by an auxiliary function that accesses and loads the data into memory.

## Possible fix
Consider publishing `ifadv` package on CRAN too. Would that make sense?

**Comment by mdingemanse** [comment]
This would indeed not be a problem if both are on CRAN but note that this bug itself is unrelated to CRAN or its submission rules: installing talkr before installing ifadv fails
